test-adapter/testadapter/tinktestrunner/Injector.hx

```diff
@@ -5,6 +5,24 @@ import haxe.macro.Context;
 import haxe.macro.Expr;
 
 class Injector {
+	public static function buildCase():Array<Field> {
+		var fields = Context.getBuildFields();
+		for (field in fields) {
+			switch (field.name) {
+				case "shouldRun":
+					field.name = "__" + field.name;
```

Comment:
I would prefer to use PatchTools to replace the existing field's expression rather than rename it. https://github.com/vshaxe/haxe-test-adapter/blob/master/test-adapter/testadapter/PatchTools.hx

Reply:
Well it doesn't cater for brace-less functions?

Reply:
That shouldn't be an issue in this case? https://github.com/haxetink/tink_testrunner/blob/6de225237698f0291b3c2dcbcf6b5c4217346e59/src/tink/testrunner/Case.hx#L8

Reply:
But it still won't let me retrieve the original expr. Because I just wanted to add a `&&` clause to it

Reply:
Why not just add if an if/return guard at the start like we do for other frameworks?